spec/departure/connection_details_spec.rb

```diff
@@ -72,8 +72,7 @@
     end
 
     context 'when the password contains bash incompatible characters' do
-      let(:env_var) { {} }
-      let(:connection_data) { { password: '!#/PASSWORD!!!' } }
+      let(:env_var) { { PERCONA_DB_PASSWORD: '!#/PASSWORD!!!' } }
```

Comment:
Would you mind letting me know the reason behind this change?

Reply:
This is an attempt to fix the failing test from master
```Failure/Error: it { is_expected.to include('-***') }
       expected "-h \"localhost\" -P 3306 -u root -*** " to include "-***"
     # ./spec/departure/connection_details_spec.rb:77
```
It looks like the password does not get recognized if is passed through the `connection_data`
Specifying the password as in the test above as `PERCONA_DB_PASSWORD` in the `env_var` seem to work